### PR TITLE
Remove test that cannot be run except on one specific developer machine

### PR DESCRIPTION
**Pull request by ndevenish**

### CODE REVIEW

**Comment by phyy-nx** [comment]
This test was added in #118 and seems important.  @graeme-winter? Thoughts?  Is it not running on Jenkins?

**Comment by Anthchirp** [comment]
No, Jenkins does not have access to that file

File: tests/model/test_experiment_list.py [hunk 1]
```diff
@@ -908,35 +908,6 @@ def test_experimentlist_from_file(monkeypatch, dials_regression, tmpdir):
     assert exp_list[0].beam
 
 
-def test_experimentlist_imagesequence_stills():
-    filenames = [
-        "/Users/rjgildea/tmp/118/Puck3_10_1_000%i.cbf.gz" % i for i in range(1, 4)
```
Comment (graeme-winter) [inline tests/model/test_experiment_list.py:913]: Totally agree that this test is useless as stands.

If this test is important, can we either:
 - move _this_ data into `dials_data` - 3 images, reasonable
 - refactor to use existing data from `dials_data`

File: tests/model/test_experiment_list.py [hunk 2]
```diff
@@ -908,35 +908,6 @@ def test_experimentlist_from_file(monkeypatch, dials_regression, tmpdir):
...
-    ]
-    for f in filenames:
-        if not os.path.exists(f):
-            pytest.skip("%s does not exist" % f)
-    experiments = ExperimentListFactory.from_filenames(filenames)
```
Comment (graeme-winter) [inline tests/model/test_experiment_list.py:918]: ... et seq.

I would expect that we have this protocol tested elsewhere.

**Review by graeme-winter — APPROVED** [review]
Yeh, test which works on (0, 1) machines (`tmp` <-) does not add value. 

If we want a test like this, should open issue to reimplement with existing data.

**Comment by ndevenish** [comment]
This would also fix #229 (this test was added by 1b793ec7a9080706085a415b6d2ae69feb2fbc5f)

File: tests/model/test_experiment_list.py (hunk 1 above)
Comment (Anthchirp) [reply tests/model/test_experiment_list.py:913]: :arrow_right: #353

**Pull request closed without merging**